src/content/configuration/ischromatic.md

```diff
@@ -7,91 +7,110 @@ sidebar: { order: 9 }
-
-## With an environment variable
-
-If you're working with Storybook 7.0 or later, you can also adjust your `chromatic` script and add the `IS_CHROMATIC` environment variable to allow you to control the story's behavior when rendered in Chromatic.
-
-```json
+```json title="package.json"
 {
   "scripts": {
     "chromatic": "IS_CHROMATIC=true chromatic"
-  }
+ }
 }
 ```
 
-Then in your component story file, set the options (e.g., [args](https://storybook.js.org/docs/writing-stories/args), [parameters](https://storybook.js.org/docs/writing-stories/parameters)) based on the environment variable as follows:
+Then, in your tests, you can check for the `IS_CHROMATIC` environment variable and set the available options accordingly (e.g., [args](https://storybook.js.org/docs/writing-stories/args), [parameters](https://storybook.js.org/docs/writing-stories/parameters)).
 
-```js
-// MyComponent.stories.js|jsx
+```ts title="MyComponent.stories.ts|tsx"
+// Adjust this import to match your framework (e.g., nextjs, vue3-vite)
+import type { Meta, StoryObj } from "@storybook/your-framework";
 
 import { MyComponent } from "./MyComponent";
 
-export default {
+const meta: Meta<typeof MyComponent> = {
   component: MyComponent,
   title: "MyComponent",
 };
 
-export const StoryName = {
+export default meta;
+type Story = StoryObj<typeof MyComponent>;
+
+export const Default: Story = {
   args: {
-    label: process.env.IS_CHROMATIC ? `I'm in Chromatic` : `Not in Chromatic`,
+    label: process.env.IS_CHROMATIC ? "I'm in Chromatic" : "Not in Chromatic",
   },
 };
 ```
 
 <div class="aside">
 
-ℹ️ For Vite-based environments, you may be required to adjust your story to allow it to access the environment variable. See the [Vite documentation](https://vitejs.dev/guide/env-and-mode.html) for more information.
+ℹ️ For Vite-based environments, you may be required to adjust your test to allow it to access the environment variable. See the [Vite documentation](https://vitejs.dev/guide/env-and-mode.html) for more information.
+
+</div>
+
+## Using `isChromatic` in your application
+
+Under specific circumstances, your components may require different behavior when running in Chromatic (e.g., [disabling animations](/docs/animations/#javascript-animations), lazy loading). In these cases, you can use `isChromatic` directly in your components to control their behavior when tested. However, this approach can lead to unexpected behavior. We recommend using `isChromatic` only in your tests or via configuration to ensure a clear separation of concerns and make the code agnostic to the environment in which it's running.
+
+<div class="aside">
+
+ 💡 If you're attempting to make code-specific changes in your project with `isChromatic`, the Chromatic package must be installed as a dependency instead of a development dependency.
 
 </div>
+
+---
+
+## Frequently asked questions
```

Comment:
@jonniebigodes I would move this up top, just after the intro paragraph, in a callout

Reply:
Will do, appreciate the feedback.